src/Azure.Functions.Cli/Common/TemplatesManager.cs

```diff
@@ -35,16 +34,18 @@ private static async Task<IEnumerable<Template>> GetTemplates()
         {
             var extensionBundleManager = ExtensionBundleHelper.GetExtensionBundleManager();
 
-            string templatesJson;
+            string templatesJson = null;
             if (extensionBundleManager.IsExtensionBundleConfigured())
             {
                 var contentProvider = ExtensionBundleHelper.GetExtensionBundleContentProvider();
                 templatesJson = await contentProvider.GetTemplates();
             }
-            else
+
+            if(string.IsNullOrEmpty(templatesJson))
             {
                 templatesJson = GetTemplatesJson();
             }
+
```

Comment:
@soninaren Is this an acceptable workaround for bundles vs non-bundles templates ?

Reply:
Not sure what changed here. Instead of else it would go in the if block the endresult here still remains the same.

Did you want to merge the custom handler templates to templates offered by bundles?  The way it is set up in the PR right now, the custom handler template would only show up for customer who does not have bundles configured.

Reply:
Discussed this offline. This does a filtration based on language which is why this works right now. Will change the code to make this more explicit.